code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx

```diff
@@ -84,14 +90,57 @@ const OrdersTableBody = ({ order, onStatusChange }: OrderTableBodyProps) => {
     </AppSelect>
   );
 
+  const handleIsPaidChange = () => {
+    !isPaid && onIsPaidChange(true);
+  };
+
+  const emailField = (
+    <AppTypography variant="caption" className="spa-order-table__body-email">
+      {email}
+    </AppTypography>
+  );
+
+  const notPaidOrderCheckbox =
+    orderStatus !== orderDeliveryStatuses.Canceled ? (
+      <AppTooltip
+        followCursor
```

Comment:
Why did you decide to use `followCursor` here

Reply:
I tried different cursor positions, but they all was unsuitable, as they were either pushed out of the order field or covered by other information.
This option looks the most user-friendly, in my opinion